### ISSUE TITLE
grpc.default.credentials.createSsl() type error

### ISSUE THREAD

**Issue by noerw** [TheThingsNetwork/node-app-sdk#68]
```js
import * as ttn from 'ttn' // ttn@2.2.2
app = await ttn.application('my-app', 'mytoken') // my-app is registered at public eu handler
```
results in `TypeError: createSsl's first argument must be a Buffer`

When running `Buffer.from(certificate, 'utf8')` in the ApplicationClient constructor I get `Could not get common name of subject from certificate.`

**Comment by romeovs** [comment]
Thanks for the report! I'm looking into it.

**Comment by romeovs** [comment]
@noerw This should be fixed in`2.2.4`, can you confirm?

**Comment by noerw** [comment]
indeed, thanks!
